ui/src/app/pages/admin/user/admin-user-table.tsx

```diff
@@ -152,6 +154,18 @@ export const AdminUserTable = withUserProfile()(
       }
     }
 
+    dataAccessContents(user: AdminTableUser): JSX.Element {
+      return (
+        <FlexRow style={{ justifyContent: 'space-evenly' }}>
+          {user.accessTierShortNames.length > 0
+            ? user.accessTierShortNames.map((accessTierShortName) => (
+                <TierBadge {...{ accessTierShortName }} />
+              ))
+            : 'N/A'}
```

Comment:
This logic is almost right ... but I see that the story doesn't call this out explicitly and it probably should have.

"N/A" is for the case when the user is **disabled** and this means that we don't calculate their appropriate access tiers.

We know when the user is Enabled-but-no-tiers, so we can show something different here.  Maybe "no access" like we have in AdminUserProfile.